### DISCUSSION TITLE
All types of Notifications

### DISCUSSION THREAD

**Discussion by agneevX** [community/community#6577] [Mobile]
Currently, I only receive notifications if someone explicitly mentions me.

I'd like to receive all notifications, that otherwise show up on github.com.

**Comment by aconsuegra** [comment]
Hi @agneevX,
thanks for getting in touch and about your feedback! GitHub Mobile currently support 4 types of push notifications, as you can see in this screenshot:

<img width="300" alt="Screen Shot 2021-10-29 at 16 59 10" src="https://user-images.githubusercontent.com/1814882/139457604-a469ea6f-095b-462b-902c-66695f41ab72.png">

You can enable/disable them from the app settings. And we will continue to bring support for more push notifications types in the future!
